frontend/src/containers/ProfilePage/ProfilePage.js

```diff
@@ -0,0 +1,159 @@
+          meal: 'SNACK',
+          ingredients: [
+            '12 eggs',
+            '2 Tbsp mustard',
+            '3 Tbsp mayonnaise',
+            '1 tsp paprika',
+            '6 cornichons, diced',
+            'Chives',
+          ],
+          directions: [
+            'Boil the eggs',
+            'Slice in half an remove yolks',
+            'Mix yolks with other ingredients except chives',
+            'Fill eggs with yolk mixture, garnish with paprika and chives',
+          ],
+          difficulty: 'MEDIUM',
+          duration: 30,
+          servings: 4,
+          photos: [
+            '/recipe-seeds/deviledeggs.jpg',
+          ],
+          created_at: '2019-05-02T00:21:41.142Z',
+          updated_at: '2019-05-02T00:21:41.142Z',
+          user_id: 1,
+        },
+      ],
+    };
+
+    this.toggle = this.toggle.bind(this);
+  }
+
+  toggle(tab) {
+    const { activeTab } = this.state;
+
+    if (activeTab !== tab) {
+      this.setState({
+        activeTab: tab,
+      });
+    }
+  }
+
+  render() {
+    const { activeTab, recipes } = this.state;
+
+    return (
+      <div className="profile-page container-fluid">
+        <div className="row">
+          <div className="profile-header">
+            <Header showSearchBar />
+          </div>
+        </div>
+        {/* Component for top user info and bio section */}
+        <UserInfo />
+        {/* Bottom section with recipes and settings */}
+        <div className="row user-panel">
+          <div className="col-md-2 user-nav-bar">
+            {/* Left navigation tabs */}
+            <Nav tabs>
+              <NavItem className={activeTab === 'recipes' ? 'user-nav-tab-active' : ''}>
+                <NavLink
+                  className={activeTab === 'recipes' ? 'recipes' : ''}
+                  onClick={() => {
+                    this.toggle('recipes');
+                  }}
+                >
+                  My Recipes
+                </NavLink>
+              </NavItem>
+              <NavItem className={activeTab === 'settings' ? 'user-nav-tab-active' : ''}>
+                <NavLink
+                  className={activeTab === 'settings' ? 'settings' : ''}
+                  onClick={() => {
+                    this.toggle('settings');
+                  }}
+                >
+                  Settings
+                </NavLink>
+              </NavItem>
+            </Nav>
+          </div>
+          <div className="col-md-9 right-pane">
+            <TabContent activeTab={activeTab}>
+              {/* Recipes tab */}
+              <TabPane tabId="recipes">
+                <div id="cards-wrapper">
+                  {recipes.map(recipe => (
+                    <RecipeCard
+                      {...recipe}
+                      key={recipe.id}
+                    />
+                  ))}
+                </div>
+              </TabPane>
+              <TabPane tabId="settings">
+                {/* Component for settings tab */}
```

Comment:
I like the spirit behind commenting, but this might be a little redundant considering the next line is literally a component called `SettingsTab`. :)

Reply:
Where I might put a comment (and maybe a newline above it) is line 131, referencing that the actual content is down here in this div.

Reply:
ha very true, I'll take that out!